app/src/main/res/layout/view_compose_schedule.xml

```diff
@@ -23,7 +23,7 @@
         android:paddingBottom="16dp"
         android:textColor="?android:textColorTertiary"
         android:textSize="?attr/status_text_medium"
-        app:drawableTint="?android:attr/textColorTertiary"
+        app:drawableTint="?attr/colorPrimary"
```

Comment:
This is a textview that is clickable?

Reply:
Yes, clicking anywhere on the textview allows the user to reschedule the post.

Arguably all the text should be colorPrimary to indicate that the whole thing is clickable, but then there's quite a lot of blue in a small area and I found it distracting.